### ISSUE TITLE
Vehicle Eligibility list 

### ISSUE THREAD

**Issue by r-woosley** [TeleportXYZ/TRIP-Guides#55]
### What's missing from the TRIP Guides

**Vehicle Eligibility List** 
It is important to share with potential driver which vehicles are approved on the platform. This guide can evolve as new ride products are added. When looking at the FAQs it says "Use an eligible 4-door vehicle". Is this list already sourced internally?

### Gist of expected content

**Ride Product** 
- Make
- Model 
- Year 

### Gist of current content

No content present on this at the moment.

**Comment by Alejandro1954-ops** [comment]
Agreed. And you may as well copy what Uber and Lyft already [have](https://www.uber.com/us/en/drive/requirements/vehicle-requirements/?_csid=hk5eREEO2WimrTdgsoidMQ&city=miami&effect=&state=oyjqzXRg6oV9_xOZXPZ2uR8xPQz190IT3nmM6x94zqE%3D) to simplify the process.

**Comment by hwhelchel** [comment]
@ivandiku what are our current vehicle eligibility requirements?
